qt/widgets/common/src/DataProcessorUI/TwoLevelTreeManager.cpp

```diff
@@ -111,6 +111,12 @@ std::vector<Command_uptr> TwoLevelTreeManager::publishCommands() {
   return commands;
 }
 
+void TwoLevelTreeManager::invalidateAllProcessed() {
+  for (auto i = 0; i < m_model->rowCount(); i++) {
+    setProcessed(false, i);
+  }
```

Comment:
I think this needs to loop through the group/row hierarchy in a similar way to [here](https://github.com/mantidproject/mantid/blob/master/qt/widgets/common/src/DataProcessorUI/QTwoLevelTreeModel.cpp#L221)